### ISSUE TITLE
Uri.of_jstr raises exceptions *in the ocaml console*

### ISSUE THREAD

**Issue by dbuenzli** [dbuenzli/brr#21]
E.g.

```ocaml
# Uri.of_jstr (Jstr.v "bla");;
Exception: TypeError: Failed to construct 'URL': Invalid URL
```

**Comment by dbuenzli** [comment]
Actually the call is guarded [for exception](https://github.com/dbuenzli/brr/blob/master/src/brr.ml#L892-L893).  This simple test prints:

```ocaml
let () =
  let is_err = Result.is_error (Uri.of_jstr (Jstr.v "bla")) in
  if is_err then Console.(log [str "is_err"]);
  ()
```

This rather looks like a problem with the exception handler business in the OCaml console. Needs more investigation. Maybe because  #2.

**Comment by dbuenzli** [comment]
We are indeed fighting with `js_of_ocaml`'s [`Callback.register_exception`](https://github.com/ocsigen/js_of_ocaml/blob/cde86b2cba215344a6f81b1db28b922b80c7b55a/lib/js_of_ocaml/js.ml#L672).  The poke object [initialisation](https://github.com/dbuenzli/brr/blob/8f704eb478cba43e4ff8157ad5689ad6762d0ad1/src/poke/brr_poke.ml#L27-L29) uses `Js_of_ocaml.Sys_js` which depends on  `Js_of_ocaml.Js` and the latter registers its own exception.

Let's re-register our exception for now. But that is brittle, the real fix is #2.
